Sources/PriorityQueue/PriorityQueue+Sequence.swift

```diff
@@ -0,0 +1,45 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift Collections open source project
+//
+// Copyright (c) 2021 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+//
+//===----------------------------------------------------------------------===//
+
+extension PriorityQueue:Sequence{
```

Comment:
Nit: please follow the coding style established in the rest of this project. We like to put spaces around punctuation characters like `:` and infix operators, and before opening braces. Keeping this consistent helps readability.